src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -0,0 +1,198 @@
+.pf-c-tree-view {
+  --pf-c-tree-view--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-tree-view--PaddingBottom: var(--pf-global--spacer--sm);
+
+  // Link
+  --pf-c-tree-view__link--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__link--Color: var(--pf-global--Color--100);
+  --pf-c-tree-view__link--m-current--Color: var(--pf-global--link--Color);
+  --pf-c-tree-view__link--m-current--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-tree-view__link--m-first--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-tree-view__list--m-nested-level-1__list-item__link--PaddingLeft: calc(var(--pf-c-tree-view__link--PaddingLeft) + var(--pf-global--spacer--md));
+  --pf-c-tree-view__list--m-nested-level-2__list-item__link--PaddingLeft: calc(var(--pf-c-tree-view__list--m-nested-level-1__list-item__link--PaddingLeft) + var(--pf-global--spacer--lg) + var(--pf-global--spacer--sm));
+  --pf-c-tree-view__list--m-nested-level-3__list-item__link--PaddingLeft: calc(var(--pf-c-tree-view__list--m-nested-level-2__list-item__link--PaddingLeft) + var(--pf-global--spacer--md) + var(--pf-global--spacer--sm));
+  --pf-c-tree-view__list--m-nested-level-1__list-item--m-expandable__link--PaddingLeft: calc(var(--pf-c-tree-view__link--PaddingLeft) + var(--pf-global--spacer--md));
+  --pf-c-tree-view__list--m-nested-level-2__list-item--m-expandable__link--PaddingLeft: calc(var(--pf-c-tree-view__list--m-nested-level-1__list-item--m-expandable__link--PaddingLeft) + var(--pf-global--spacer--md));
+  --pf-c-tree-view__list--m-nested-level-3__list-item--m-expandable__link--PaddingLeft: calc(var(--pf-c-tree-view__list--m-nested-level-2__list-item--m-expandable__link--PaddingLeft) + var(--pf-global--spacer--md));
+
+  // link hover/focus
+  --pf-c-tree-view__node--hover--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-tree-view__link--focus--BackgroundColor: var(--pf-global--palette--black-150);
+
```

Comment:
Are the node:hover, and link:focus colors different on purpose? The difference is nearly indiscernible (`#f5` vs `#f0`), I'm wondering if they should be the same. And if they should be triggered on the same element, using `:focus-within` if the element changing background color isn't focusable.